src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stylesObj = {};
+  const lines = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy before each operation. The split method does not mutate the original string, so you can chain the necessary methods directly after splitting.